locales/templates/app.pot

```diff
@@ -0,0 +1,647 @@
+msgid "Cache"
+msgstr ""
+
+#. Download list
+msgctxt "no_downloads"
+msgid "No downloads"
+msgstr ""
+
+msgctxt "download_started"
+msgid "Downloading…"
+msgstr ""
+
+msgctxt "download_file_not_supported"
+msgid "File type not supported"
+msgstr ""
+
+msgctxt "cannot_find_the_file"
+msgid "Cannot find the file."
+msgstr ""
+
+msgctxt "cannot_delete_the_file"
+msgid "Cannot delete the file."
+msgstr ""
+
+#. Download status
+#. File download status is paused
+msgctxt "pause"
+msgid "Paused"
+msgstr ""
+
+#. File download status is pending
+msgctxt "pending"
+msgid "Pending"
+msgstr ""
+
+#. File download status is running
+msgctxt "running"
+msgid "Downloading"
+msgstr ""
+
+#. File download status is successful
+msgctxt "successful"
+msgid "Successful"
+msgstr ""
+
+#. File download status is failed
+msgctxt "failed"
+msgid "Failed"
+msgstr ""
+
+#. File download status is unknown
+msgctxt "unknown"
+msgid "Unknown"
+msgstr ""
+
+#. Setting data saving path dialog
+msgctxt "setting_dialog_sd_card"
+msgid "SD card"
+msgstr ""
+
+msgctxt "setting_dialog_internal_storage"
+msgid "Internal storage"
+msgstr ""
+
+#. BrowsingHistoryActivity
+msgctxt "browsing_history_clear_btn"
+msgid "CLEAR BROWSING HISTORY"
+msgstr ""
+
+msgctxt "browsing_history_empty_view_msg"
+msgid "No history"
+msgstr ""
+
+msgctxt "browsing_history_date_today"
+msgid "Today"
+msgstr ""
+
+msgctxt "browsing_history_date_yesterday"
+msgid "Yesterday"
+msgstr ""
+
+msgctxt "browsing_history_menu_delete"
+msgid "Delete"
+msgstr ""
+
+msgctxt "browsing_history_dialog_confirm_clear_message"
+msgid "Clear all browsing history?"
+msgstr ""
+
+msgctxt "browsing_history_dialog_btn_clear"
+msgid "Clear"
+msgstr ""
+
+#. Remove the item from Download file panel
+msgctxt "remove_from_list"
+msgid "Remove from list"
+msgstr ""
+
+#. Remove the item from Download file panel and delete the file
+msgctxt "delete_file"
+msgid "Delete file"
+msgstr ""
+
+#. Long Screenshot
+msgctxt "screenshot_saved"
+msgid "Screenshot saved"
+msgstr ""
+
+msgctxt "screenshot_failed"
+msgid "Failed to capture screenshot"
+msgstr ""
+
+msgctxt "screenshot_illustration"
+msgid "capturing screenshot of entire page…"
+msgstr ""
+
+#. Snack bar message
+msgctxt "message_fallback_save_to_primary_external"
+msgid "No SD card detected. File was saved to internal storage."
+msgstr ""
+
+msgctxt "message_start_to_save_to_removable_storage"
+msgid "File saved to SD card"
+msgstr ""
+
+msgctxt "message_removable_storage_space_not_enough"
+msgid "SD card is almost full. File was saved to internal storage."
+msgstr ""
+
+#. ScreenshotGridFragment
+msgctxt "screenshot_grid_empty_text_title"
+msgid "No screenshots"
+msgstr ""
+
+#. The element between this prefix and postfix is a button image, not a
+#. string.         So we cannot use the placeholder here.
+msgctxt "screenshot_grid_empty_text_msg_prefix"
+msgid "Tap "
+msgstr ""
+
+msgctxt "screenshot_grid_empty_text_msg_postfix"
+msgid ""
+"  on the menu to capture\n"
```

Comment:
Add comment: "This string is preceded by the image of the button, keep the leading whitespace. Final result: 'Tap [image] on the menu to capture an entire web page'.